### PR TITLE
Clarify a couple minor points

### PR DESCRIPTION
**Pull request by fw-immunant**

### CODE REVIEW

File: src/basic-syntax/static-and-const.md [hunk 1]
```diff
@@ -39,9 +39,11 @@ fn main() {
 ```
 
 As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and the variable lives through the entirety of the program execution.
+When a globally-scoped value does not have a reason to need its own address in memory, `const` is generally preferred.
 
 
 We will look at mutating static data in the [chapter on Unsafe Rust](../unsafe.md).
+Because `static` variables are accessible from any thread, mutable static variables require manual, unsafe, synchronization of accesses.
```
Comment (qwandor) [inline src/basic-syntax/static-and-const.md:46]: This could link to https://google.github.io/comprehensive-rust/unsafe/mutable-static-variables.html which talks about mutable statics a bit more.

File: src/basic-syntax/static-and-const.md [hunk 2]
```diff
@@ -39,9 +39,11 @@ fn main() {
 ```
 
 As noted in the [Rust RFC Book][1], these are not inlined upon use and have an actual associated memory location.  This is useful for unsafe and embedded code, and the variable lives through the entirety of the program execution.
+When a globally-scoped value does not have a reason to need its own address in memory, `const` is generally preferred.
```
Comment (qwandor) [inline src/basic-syntax/static-and-const.md:42]: ... and doesn't have interior mutability. E.g. making a static `Mutex<MyType>` into a const wouldn't work.

**Comment by henrif75** [comment]
Hey @fw-immunant thank you for your suggestions. Would you like to check @qwandor 's comments before we're able to merge your PR?

File: src/basic-syntax/static-and-const.md (hunk 2 above)
Comment (fw-immunant) [reply src/basic-syntax/static-and-const.md:42]: Would saying "object identity" instead of "its own address in memory" capture both of these requirements, and then we could specify "(e.g. requiring its own address in memory or its own dynamic state such as types like `Mutex<T>` which have interior mutability)"? Or is that too wordy? Maybe the latter bit could go in presenter notes.

File: src/basic-syntax/static-and-const.md (hunk 1 above)
Comment (fw-immunant) [reply src/basic-syntax/static-and-const.md:46]: Maybe it would make more sense to change the link in the previous sentence:
```
We will look at [mutating static data](../unsafe/mutable-static-variables.md) in the chapter on Unsafe Rust.
```

File: src/basic-syntax/static-and-const.md (hunk 2 above)
Comment (qwandor) [reply src/basic-syntax/static-and-const.md:42]: Sounds reasonable.

File: src/basic-syntax/static-and-const.md (hunk 1 above)
Comment (qwandor) [reply src/basic-syntax/static-and-const.md:46]: Sure, go for it.

**Comment by fw-immunant** [comment]
Pushed an update and also clarified the first sentence of the consts/statics page which implied that `const`s relate to global *state*, which is specifically false.

**Review by qwandor — APPROVED** [review]
